site/learn/Learn-Schema.md

```diff
@@ -311,6 +311,7 @@ In this case, if you query a field that returns the `SearchResult` union type, y
 # { "graphiql": true}
 {
   search(text: "an") {
+    __typename
```

Comment:
Can we add something to the text regarding this?

Reply:
Sure thing!

Reply:
@stubailo That should be all set now.